### PR TITLE
Formlabel counter

### LINKED ISSUE #2219: [Textfield & Textarea] Counter
Capture d’écran 2023-08-22 à 15 12 43

We need to cover Textfield & Textarea counter on `form-field` side.
A few detail must be discussed with design (typing behaviors, digits colors over defined percentage, etc.)

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
## Description

fix #2219 

-----

-----

Capture d’écran 2024-01-10 à 11 42 08

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2500/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/11660/ui-diff)

**Review by ymalie-lucca — APPROVED** [review]

File: packages/scss/src/components/formLabel/component.scss [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
-  display: inline-block;
-  font-size: var(--components-formLabel-fontSize);
-  line-height: var(--components-formLabel-lineHeight);
-  width: fit-content;
+	color: var(--components-formLabel-color);
+	display: flex;
+	font-size: var(--components-formLabel-fontSize);
+	line-height: var(--components-formLabel-lineHeight);
+	width: var(--components-formLabel-width);
+	gap: var(--spacings-XS);
```
Comment (jeremie-lucca) [inline packages/scss/src/components/formLabel/component.scss:7]: This gap generate spacing issues with tootltip "?" icon and mandatory's "*".
image

**Review by jeremie-lucca — CHANGES_REQUESTED** [review]
Also, I'm pretty sure that @Supamiu will require to have this value inside the textfield component itself (which would be annoying for us). If this is the case, maybe we should discuss the possibility to move the counter as a field affix with Clément.

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2500/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/13993/ui-diff)

**Comment by Supamiu** [comment]
The angular implementation for this component is now done 🙂

**Review by Supamiu — DISMISSED** [review]

File: packages/ng/form-field/form-field.translate.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+export const luFormFieldTranslations: ILuTranslation<LuFormFieldTranslations> = {
+	fr: {
+		counter: 'Votre publication fait {{current}} caractères de long. {{max}} caractères maximum sont autorisés.',
+	},
+	en: {
+		counter: 'Your message is {{current}} characters long. A maximum of {{max}} is allowed.',
+	},
+};
```
Comment (GuillaumeNury) [inline packages/ng/form-field/form-field.translate.ts:19]: Could we add all langs? Copilot do it pretty easily

Comment (Supamiu) [reply packages/ng/form-field/form-field.translate.ts:19]: Oh I'm not using copilot yet so yeah I didn't translate for other langs

Comment (vvalentin-lucca) [reply packages/ng/form-field/form-field.translate.ts:19]: Small detail, but it would be even cooler with the management of plurals here.

Comment (Supamiu) [reply packages/ng/form-field/form-field.translate.ts:19]: oof ! I think this might be out of scope but we can start to discuss it for sure.

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2500/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/15162/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2500/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2500/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/15243/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2500/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/15247/ui-diff)

**Review by jeremie-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2500/storybook

**Review by jeremie-lucca — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/15261/ui-diff)

**Pull request merged**